index.html

```diff
@@ -0,0 +1,134 @@
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+                <li><a href="" id="cc-link">F1Campaign</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <section class="intro">
+        <p id="hello">"Hello!,Sharing world"</p>
+        <div class="title-intro">
+            <h1>FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="p-intro">
+            <p class="summit-intro">Welcome to the 7th global summit of Formula 1 fans.
+                Celebrate,have fun and bless someone.Venue:Vienna,Austria. </p>
+        </div>
+        <div class="date">
+            <p class="show-date">2023.9.8(FRI) ~ 9(SAT)</p>
+            <p>@Grand Prix of Vienna and more</p>
+        </div>
+    </section>
+
+    <section class="program">
+        <h2>Main program</h2>
+        <hr class="sm-line">
+
+        <div id="gr-program">
+            <div class="grid-program">
+                <img src="images/program_icon_01 1.svg" alt="lecture-icon" class="program-icon">
+                <p class="program-points">Lecture</p>
+                <p class="program-ex">Listen to the speakers from various countries 
+                    about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_02 1.svg" alt="exhibition-icon" class="program-icon">
+                <p class="program-points">CC Exhibition</p>
+                <p class="program-ex">Appreciate various creations applying 
+                    CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_03 1.svg" alt="forum-icon" class="program-icon">
+                <p class="program-points">Forum</p>
+                <p class="program-ex">Have the time to share your thoughts and 
+                    opinions with experts for each topic.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_04 1.svg" alt="workshop-icon" class="program-icon">
+                <p class="program-points">Workshop</p>
+                <p class="program-ex">Try creating your own work using open source license 
+                    rather than just watching at it.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_05 1.svg" alt="ignite-icon" class="program-icon">
+                <p class="program-points">CC ignite</p>
+                <p class="program-ex">get opportunities to network with CC affiliates around the world, 
+                    also after the summit.</p>
+            </div>
+        </div>
+
+        <p id="see-program1">SEE THE WHOLE PROGRAM</p>
```

Comment:
- [optional] please kindly ensure you follow the original design's color. Instead of `#EC5242` you are using a different color. Kindly fix this for all sections. If possible, increase the height of the join button as well. Nice work
- yours
![Screenshot (1300)_LI](https://github.com/MarkYegon7/Capstone-Project/assets/65298048/9ebd122a-cbc4-4a87-a8fa-c84acd5cbb61)
original
![Screenshot (1301)](https://github.com/MarkYegon7/Capstone-Project/assets/65298048/79b6c831-cc4c-497f-875c-dc5104ae5567)

Reply:
Used the required color.
I've increased the height of the join button.

Reply:
Used the required color.
I've increased the height of the join button.